resources/views/home.blade.php

```diff
@@ -1,51 +1,68 @@
 @extends('layouts.app')
 
+@section('title', 'Home')
+@section('page-title', 'Home')
+
 @section('content')
 <div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
-            <div class="card">
-                <h3>{{ Auth::user()->name }}</h3>
-                
-                <div class="card-header">Dashboard for User-ID: {{ Auth::user()->id }}</div>
-
-                <div class="card-body">
-                    You are logged in!
-                </div>
-                <div>
-                    <a href="{{ route('sons.create') }}" >Add Statement of Need</a>
-                </div>
+
+    <div class="view cwd-component cwd-card-grid">
+        <div class="views-row card">
+            <div class="field title">
+                <h3><a href="#" class="designer"><span class="deco">Welcome:<br>{{ Auth::user()->name }}</span></a></h3>
+            </div>
+        </div>
+
+        <div class="views-row card">
+            <div class="field link">
+                <h4>Get Started:</h4>
+                <a class='link-button' href="{{ route('sons.create') }}" ><i class="la la-plus"></i> Add Statement of Need</a>
             </div>
         </div>
+
+        </div>
     </div>
-    
+
     @if(Auth::user()->hasRole('admin'))
         <table class="table bordered">
             <caption>Statements Of Need List</caption>
             <thead>
                 <tr>
-                    <th></th>
-                    <th scope="col">Submitted date</th>
-                    <th scope="col">Title</th>
-                    <th scope="col">Submitted by</th>
+                    <th scope="col"> @sortablelink('title', 'Title') </th>
+                    <th scope="col"> @sortablelink('user_netid', 'Submitter')</th>
+                    <th scope="col"> @sortablelink('college_name', 'College/Unit')</th>
+                    <th scope="col"> @sortablelink('status', 'Status')</th>
+                    <th scope="col"> @sortablelink('created_at', 'Ttimestamp')</th>
+                    <th scope="col">Actions</th>
                 </tr>
             </thead>
             <tbody>
                 @foreach($sons as $son)
                     <tr>
-                        <th scope="row">{{ $son->id }}</th>
-                        <td>{{ $son->created_at->format('Y-F-j')}}</td>
-                        <td>
-                            <a href="{{route('sons.show', $son->id)}}">{{$son->title}}</a> 
+                        <td><a href="{{route('sons.show', $son->id)}}">{{$son->title}}</a></td>
                         <td>{{ $son->user_netid }}</td>
+                        <td>{{ $son->college_name }}</td>
+                        <td>{{ $son->status}}</td>
+                        <td>{{ $son->created_at->format('Y-m-j')}}</td>
```

Comment:
Dates are coming out like "2020-04-9", which is weird. Just going with "2020-04-09".
```suggestion
                        <td>{{ $son->created_at->format('Y-m-d')}}</td>
```

Reply:
I was trying to keep column width as narrow as possible